ui/src/components/containers/taskBox.tsx

```
@@ -0,0 +1,111 @@
+          )}
+          <p>
+            <strong class="font-medium text-gray-300">Image:</strong>{" "}
+            {props.taskDetails.image}
+          </p>
+          <p>
+            <strong class="font-medium text-gray-300">Parameters:</strong>
+          </p>
+          <ul class="ml-4 list-disc text-gray-400">
+            {props.taskDetails.parameters &&
+              props.taskDetails.parameters.map((param) => (
+                <li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)